lib/monza/verification_response.rb

```diff
@@ -37,30 +37,16 @@ def initialize(attributes)
     end
 
     def is_subscription_active?
-      if @latest_receipt_info.last
-        if @latest_receipt_info.last.cancellation_date
-          false
-        else
-          @latest_receipt_info.last.expires_date_ms >= Time.zone.now
-        end
-      else
-        false
-      end
+      receipt = latest_active_transaction_receipt
+      receipt.present? && receipt.cancellation_date.nil?
     end
 
     def is_any_subscription_active?
-      expires_dates_ms = @latest_receipt_info.reject(&:cancellation_date).map(&:expires_date_ms)
-      latest_expires_date_ms = expires_dates_ms.max
-
-      if latest_expires_date_ms
-        latest_expires_date_ms >= Time.zone.now
-      else
-        false
-      end
+      is_subscription_active?
```

Comment:
Hey @sidonath correct, this would change behavior. This is an edge case but happened to me last year, I should've documented it more, but I believe it was due two two separate product subscriptions and one was cancelled but the subscription for the other product was still active

Reply:
@samjohn @sidonath actually I think this is fine as this method will still return true if any, not just the latest, is active